src/mSupplyMobileApp.js

```diff
@@ -223,6 +235,7 @@ class MSupplyMobileAppContainer extends React.Component {
     return (
       <View style={globalStyles.appBackground}>
         <NavigationBar
+          routeName={this.getCurrentRouteName(navigationState)}
```

Comment:
adding `currentRouteName` to props through `mapStateToProps` might be a little nicer - although you couldn't use the class method. But I don't know why it is a class method? Can combine the methods as one selector?